src/stackwalk.c

```diff
@@ -1108,6 +1108,42 @@ JL_DLLEXPORT void jl_print_backtrace(void) JL_NOTSAFEPOINT
     jlbacktrace();
 }
 
+JL_DLLEXPORT void jl_print_task_backtraces(void)
+{
```

Comment:
It might be helpful for this to take an IO stream or an in-memory buffer as an argument, so we could direct its output to e.g. a a file or HTTP response.

Not sure if that would create locking/scheduling complications.

The Go version takes a buffer as an argument, maybe for this reason: https://pkg.go.dev/runtime#Stack

Reply:
That's a good idea, yeah!

Reply:
This would need more surgery (the actual printing of backtrace entries is [a](https://github.com/JuliaLang/julia/blob/master/src/stackwalk.c#L1102) [few](https://github.com/JuliaLang/julia/blob/master/src/stackwalk.c#L657) [levels](https://github.com/JuliaLang/julia/blob/master/src/stackwalk.c#L645) [down](https://github.com/JuliaLang/julia/blob/master/src/stackwalk.c#L620)) and maybe changing the interface for the existing functions (`jl_print_backtrace()`, `jlbacktracet()`, ...).

IMO, that should be a separate PR.